migrations/2023-02-05-154242_registration_review_content/up.sql

```diff
@@ -0,0 +1,12 @@
+-- add new registration mode
+alter type registration_mode_enum add value 'review_content';
+
+-- add table for comment review queue
+create table review_comment (
+  id            serial    primary key,
+  comment_id    int       unique references comment on update cascade on delete cascade not null,
+  approved      bool      not null default false,
+  approver_id   int       references local_user on update cascade on delete cascade,   -- user who approved comment
```

Comment:
In the code, you have this reference `person`, not `local_user` . 

Based on the standard of person and comment reports, this should also reference the person table.